src/Database/Schema/Table.php

```diff
@@ -14,789 +14,9 @@
  */
 namespace Cake\Database\Schema;
 
-use Cake\Database\Exception;
-use Cake\Database\Type;
-use Cake\Datasource\ConnectionInterface;
-
 /**
- * Represents a single table in a database schema.
- *
- * Can either be populated using the reflection API's
- * or by incrementally building an instance using
- * methods.
- *
- * Once created Table instances can be added to
- * Schema\Collection objects. They can also be converted into SQL using the
- * createSql(), dropSql() and truncateSql() methods.
+ * @deprecated Provided for backward compatibility. Use TableSchema instead.
  */
-class Table
+class Table extends TableSchema
```

Comment:
![image](https://cloud.githubusercontent.com/assets/551022/14919843/ac73d4b8-0df8-11e6-8881-6c14adbf1bfe.png)

As you can see it works as I wanted. The class is indicated as deprecated on old code but does not break anything.